### DISCUSSION TITLE
mssql: Add config capability with schema inclusion/exclusion

### DISCUSSION THREAD

**Discussion by lcorneliussen** [sqldef/sqldef#631] [Feature requests]
On Azure SQL, the following view is added when exporting (or removed when applying schema without it):

    CREATE VIEW sys.database_firewall_rules A

How can sys.* be excluded? In my case I'd only like to syn dbo.*

**Comment by lcorneliussen** [comment]
I've started an impl on adding `--config` to `mssqldef`. Next up could be to support `target_schema` (just filtering everything on the schema).

Adding `target_schemas` or `skip_schemas` to the config settings would also be an option.

@k0kubun  would you like to help out? I'm not (yet) too familiar with programming go.

**Comment by k0kubun** [comment]
You may look at https://github.com/sqldef/sqldef/pull/305 for starters.

**Comment by chi-bd** [comment]
@lcorneliussen 
I believe your request has been addressed in PR #734. Could you please test it out?
```shell
$ mssqldef -U sa -P password yourdb --config-inline="skip_tables: sys\..*" --export
```
